### DISCUSSION TITLE
Cannot import new Accounts

### DISCUSSION THREAD

**Discussion by espe808** [YetiForceCompany/YetiForceCRM#14780] [How to]
Hello,
I cannot import new accounts.

There is this page loading in loop:

![Schermata 2021-06-04 alle 19 40 56](https://user-images.githubusercontent.com/85027316/120842330-d96ef700-c56c-11eb-8adb-78bd12b2e46b.png)

Then I have this error:

![Schermata 2021-06-04 alle 19 38 25](https://user-images.githubusercontent.com/85027316/120842247-bcd2bf00-c56c-11eb-8446-63e77494dd84.png)

**Comment by mariuszkrzaczkowski** [comment]
report it as a bug, it is important that the cron configuration is correct
1. attach the system configuration report
2. delete all tables on import
3. delete all logs and enable full logs
4. start the import and attach the logs
